twill11/hometask-4/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require_relative '../my_hash/my_hash'
+
+RSpec.describe MyHash do
+  describe '[]=' do
+    before do
+      my_hash['hello'] = 'world'
+    end
+
+    let(:my_hash) { described_class.new }
+
+    context 'when write key value' do
+      it { expect(my_hash.size).to eq 1 }
+    end
```

Comment:
_It_ should have a label or use specify instead of one.

Please read.
https://github.com/rubocop/rspec-style-guide#it-and-specify